modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -1541,11 +1574,18 @@ sub production_name_mapping {
 ### As the name said, the function maps the production name with the species URL, 
 ### @param production_name - species production name
 ### Return string = the corresponding species.url name which is the name web uses for URL and other code
+### Fall back to production name if not found - mostly for pan-compara
   my ($self, $production_name) = @_;
+  my $mapping_name = $production_name;
```

Comment:
is that correct this assignment? If i understand it correctly in case the function doesnt find a corresponding production_name it will return what was sent. should it not return empty or say something "production_name mapping not found"

Reply:
as the comment says, it needs to fall back to production name if there is no corresponding species url is not found. 
There should always be a species url for a production name. If in any case it didnt find one and returns a production name, will NV continue to work?

Reply:
See the comment - there's no URL configured locally for pan compara species, but we don't want this method to return undef, otherwise links to species on other sites won't work. The alternative is to put the fallback into everywhere we create pan_compara links - doable, but probably outside the scope of this PR.

Reply:
No, there won't be a species URL for pan compara species, because they're on a different website. For example you might have human homologues to bumblebee on the metazoa pan compara page, but the metazoa site doesn't know that the human url is Homo_sapiens.